### ISSUE TITLE
Improve documentation/help strings

### ISSUE THREAD

**Issue by carlpett** [martinlindhe/wmi_exporter#94]
Currently the only documentation we have (apart from the code) is at best a link to MSDN with the fields exposed by WMI. While documentation can be hard to keep up to date, I believe it would make it easier for new users to get started with the exporter. 

There are three areas I think we can improve:
- Collector docs in the repo, linked from the main README. Initially this could simply be the TYPE/HELP strings that are generated as part of the exposition format (like [memcached_exporter](https://github.com/prometheus/memcached_exporter) for example). Which brings the next point...
- We don't have help strings for quite a large number of the metrics (I'm responsible for quite a few of those myself). This is mostly drudge-work with copy-pasting from perfmon's descriptions. 
- Some getting-started stuff, useful queries, dashboards, alerts... There is a discussion about this sort of thing [here](https://github.com/prometheus/node_exporter/pull/590) and on the dev-maillist. 

Thoughts? There is of course an initial cost of doing this, but I think the benefits and the quite low upkeep cost makes it reasonable.

**Comment by martinlindhe** [comment]
This is a good idea!

> Collector docs in the repo, linked from the main README

This I think we should definitely do, at least expand with some example command lines, as was brought up in #105

> We don't have help strings for quite a large number of the metrics

As far as I can remember from "documenting" fields myself (or rather, adding the actual WMI field names"), the MS documentation can be a bit hard to parse, and sometimes we've seem to have discovered the docs aren't fully accurate. I would prefer to just refer to MS for supplying proper information, it feels to me that fully documenting it all in our end could be hard to get right.
At least my initial idea was just to leave references to the actual values we pulled, so someone could dig deeper without looking at the source code.

> Some getting-started stuff, useful queries, dashboards, alerts...

This, too I think we should do. And I also think we should maybe break it out from the README.
But how to track the docs?

* In a doc-folder in the repo
* Or n webpage using the gh-pages branch fore the repo
* Or using the github wiki?
